molSimplify/Scripts/qcgen.py

```diff
@@ -1082,7 +1082,7 @@ def molcgen(args, strfiles, method):
             mdir = rdir+'/'+nametrunc+mmd
             if not os.path.exists(mdir):
                 try:
-                    os.mkdirs(mdir)
+                    os.makedirs(mdir)
```

Comment:
Unsure if `mkdir()` or `makedirs()`. `mkdirs()` raised AttributeError that was caught by one of the bare except statements.